### PR TITLE
Update PHPStan recommended config to scanFiles

### PR DESCRIPTION
**Pull request by dennisameling**
Based on the comment in https://github.com/php-stubs/woocommerce-stubs/pull/6#issuecomment-696942193 :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Include all stubs in PHPStan configuration file.
 
 ```yaml
 parameters:
-    bootstrapFiles:
+    scanfiles:
```
Comment (szepeviktor) [inline README.md:34]:
```suggestion
    bootstrapFiles:
    # scanfiles: # If you do not extend WooCommerce classes
```

**Review by szepeviktor — CHANGES_REQUESTED** [review]

**Comment by szepeviktor** [comment]
If we tell it everybody they will use it and all those extending a WC class will **Complain**.

**Comment by dennisameling** [comment]
Makes sense, let's leave it as-is then with the PR you just merged :) https://github.com/php-stubs/woocommerce-stubs/commits/master

**Comment by szepeviktor** [comment]
Can you share where you use these stubs?

**Comment by dennisameling** [comment]
It's in a private repo for a customer's WP plugin, sorry :( thanks a lot for providing the stubs in the first place, much appreciated 😊 🚀

**Pull request closed without merging**